aleph-client/docker/subxt-integration-entrypoint.sh

```diff
@@ -1,13 +1,14 @@
 #!/usr/bin/env bash
 
-subxt codegen --derive Clone --derive Debug --derive Eq --derive PartialEq | rustfmt --edition=2021 --config-path aleph-node/rustfmt.toml > aleph_zero.rs
+subxt codegen --derive Clone --derive Debug --derive PartialEq --derive Eq \
+  | sed 's/::[ ]*subxt[ ]*::[ ]*utils[ ]*::[ ]*AccountId32/::subxt::utils::Static<::subxt::ext::sp_core::crypto::AccountId32>/g' \
```

Comment:
cli should have `substitute-type` option for that

Reply:
See [comment above](https://github.com/Cardinal-Cryptography/aleph-node/pull/1335#issuecomment-1681913775) - I could be wrong, but tried every possible path to `AccountId` I was able to come up with and had no luck

Reply:
add a jira task to fix it when we will have subxt > 0.29

Reply:
https://cardinal-cryptography.atlassian.net/browse/A0-3050

Reply:
Nevermind, I figured out how to substitute, removing that task